wolox-books/src/app/screens/auth/bookdetail/bookdetail.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, OnInit } from '@angular/core';
+import { BookServiceService } from 'src/app/services/book-service.service';
+import { ActivatedRoute} from '@angular/router';
+import { Book } from 'src/app/models/book.model';
+
+@Component({
+  selector: 'app-bookdetail',
+  templateUrl: './bookdetail.component.html',
+  styleUrls: ['./bookdetail.component.scss']
+})
+export class BookdetailComponent implements OnInit {
+
+  book: Book;
+  constructor(private bookService: BookServiceService, 
+    private route: ActivatedRoute) { }
+
+  ngOnInit() {
+    const id: string = this.route.snapshot.params.id;
+    this.bookService.getBookDetail(id)
```

Comment:
instead of subscribing use the async pipe: https://blog.angularindepth.com/angular-question-rxjs-subscribe-vs-async-pipe-in-component-templates-c956c8c0c794